Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.Core/Types/ResetPasswordEmailNotification.cs

```diff
@@ -0,0 +1,10 @@
+using VirtoCommerce.NotificationsModule.Core.Model;
+
+namespace VirtoCommerce.NotificationsModule.Core.Types
+{
+    public class ResetPasswordEmailNotification : EmailNotification
+    {
+        [NotificationParameter("Reset password URL")]
```

Comment:
This attribute redundant, we should avoid of using any descriptions at the code level.
For an available properties legend we use JSON resources in UI